src/test/java/org/nuxeo/io/fsexporter/test/DocumentBuilder.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * (C) Copyright 2018 Nuxeo SA (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     dmetzler
+ */
+package org.nuxeo.io.fsexporter.test;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.commons.lang3.StringUtils;
+import org.nuxeo.ecm.core.api.Blob;
+import org.nuxeo.ecm.core.api.CoreSession;
+import org.nuxeo.ecm.core.api.DocumentModel;
+import org.nuxeo.ecm.core.api.impl.blob.StringBlob;
+
+public class DocumentBuilder {
+    private String docType;
+
+    private String path;
+
+    private String name;
+
+    private String title;
+
+    private Blob content;
+
+    private List<Blob> additonalContent = new ArrayList<>();
+
+    public DocumentBuilder(String docType) {
+        this.docType = docType;
+    }
+
+    public static DocumentBuilder folder() {
+        return new DocumentBuilder("Folder");
+    }
+
+    public static DocumentBuilder file() {
+        return new DocumentBuilder("File");
```

Comment:
"File" could be in a Constant
Maybe there is already one

Reply:
As the constant is never used elsewhere and is what gives the method its nature, I prefer to let it here.